wiki/Help_centre/Beatmapping/en.md

```diff
@@ -39,6 +39,14 @@ If you are still having issues or want more information on this, have a read thr
 
 **The number of slots available depends on the number of ranked maps you have. For details, see [osu!supporter § Increased limits](/wiki/osu!supporter#increased-limits).**
 
+## My beatmap has been stolen or re-uploaded by another user without my consent, and I want the infringing copy to be taken down! {id=report-stolen-beatmap}
```

Comment:
From a non-mapper perspective, is it worth specifying what counts as a beatmap being stolen, or is that implicitly well known?

Reply:
I don't think it's worth it tho, since most of the time what constitutes as "stealing" can be boiled down to common sense

e.g. people copying that one dorchadas slider to be used in a different map with a different context wouldn't be considered by many as "stealing"
but sth like [this](http://www.youtube.com/watch?v=xURjkQ6pjQQ) where a mapper is trying to pass off another mapper's creation as their own would definitely fall into "stealing"

Either way in the end on borderline cases the ones that will decide whether something is considered as a valid "beatmap stealing" are the legal team themselves so yeah

Reply:
Please just remove the "stealing" term, it's stupid. The concept of "stealing" publicly available data is stupid.

```suggestion
## My beatmap/content has been re-uploaded by another user without my consent, and I want the infringing copy to be taken down! {id=report-content-dmca}
```